vent/common/values.py

```diff
@@ -300,5 +300,3 @@ def to_dict(self):
 given as argument to the ``callable_returning_boolean`` which will return
 whether (``True``) or not (``False``) a value is allowed.
 """
-
-MAX_STACK_DEPTH = 20
```

Comment:
Why have you removed this from values.py and copied to fashion.py?

Reply:
https://github.com/CohenLabPrinceton/Ventilator-Dev/pull/46#discussion_r421868860

I don't really care either way